src/Microsoft.DotNet.Build.Tasks.Feed/src/PushArtifactsInManifestToFeed.cs

```diff
@@ -219,7 +257,94 @@ public async Task<bool> ExecuteAsync()
                 Log.LogErrorFromException(e, true);
             }
 
+            if (Log.HasLoggedErrors)
+            {
+                await CreateGitHubIssue(repoUrl, commit, buildId);
+            }
+
             return !Log.HasLoggedErrors;
         }
+
+        /// <summary>
+        /// Creates a new GitHub issue.
+        /// </summary>
+        /// <param name="repoUrl">GitHub source repo IRL.</param>
```

Comment:
```suggestion
        /// <param name="repoUrl">GitHub source repo URL.</param>
```

Although I love that this implies that you can meet a repo IRL